src/components/modules/alt-heading-block/index.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  StyledAltHeadingBlockSubtitle,
+  StyledAltHeadingBlockTitle,
+  StyledAltHeadingBlockWrapper,
+} from './alt-heading-block.styles';
+import type { AltHeadingBlockProps } from './alt-heading-block.types';
+
+import type { HeadingProps } from '@/components/elements/heading/heading.types';
+
+/**
+ * @param title: This string can have highlighted text on the segments that are between \`\*content\*\`
+ */
+export const AltHeadingBlock = ({ title, subtitle, mediumHeading = false }: AltHeadingBlockProps) => {
+  const headingSize: HeadingProps['fontSize'] = mediumHeading ? 'm' : 'l';
+  const formattedTitle = title.split('`').map(segment => {
+    const isHighlighted = segment.includes('*');
+    if (!isHighlighted) {
+      return segment;
+    }
+    return <span>{segment.replaceAll('*', '')}</span>;
+  });
```

Comment:
Can we move this to a helper utility and reuse it where is needed?. We are using the same logic also in `HeadingBlock`